refactor(Icons): Replaces custom icons with Ant Design 5 icons

EnxDev opened the pull request:
<!---
Please write the PR title following the conventions at https://www.conventionalcommits.org/en/v1.0.0/
Example:
fix(dashboard): load charts correctly
-->

### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
This PR improves the overall standardization and maintainability of the project by replacing custom icons with Ant Design 5 icons. 
It removes nearly all FontAwesome icons, streamlining the icon set to ensure consistency. 
The Icon component has been enhanced to support new features and improved performance. 

Additionally, missing icons have been added, and standardized sizing has been applied across various icons, including those used in buttons, to ensure a cohesive UI. 

Finally, it eliminates direct imports of icons from Ant Design, simplifying the import structure and improving code readability.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
<!--- Skip this if not applicable -->
- Before
![Screenshot 2025-03-07 095627](https://github.com/user-attachments/assets/de5d4416-5eb6-41a1-89f1-fda289a79b5f)

- After
![Screenshot 2025-03-07 095923](https://github.com/user-attachments/assets/7f949305-09e9-440e-bc4e-439c8cbe8f69)

Tags :

- before
![before](https://github.com/user-attachments/assets/c4f03174-a4bb-4417-98ce-298aae5154a0)
- after
![after](https://github.com/user-attachments/assets/b56432bc-de98-40fd-bfe0-06b9671d6bf1)

Filter: 

- before
![before](https://github.com/user-attachments/assets/d5f645b3-908b-4447-9876-ede46f589e3e)
- after
![after](https://github.com/user-attachments/assets/8bfee170-7caa-4a3e-924d-163358ff024e)
- before
![before-h](https://github.com/user-attachments/assets/9420186a-5d48-41ee-89bf-4aacf26dccdc)
- after
![after-h](https://github.com/user-attachments/assets/febde644-2ebd-4c2f-bd26-dbc513e80251)
- before
![before-filter](https://github.com/user-attachments/assets/a0ad1a1d-5e9d-4670-8767-2dfbc5140e22)
- after
![after-filter](https://github.com/user-attachments/assets/a56c2119-7e92-4c71-9ee2-a3a1eb6d9c40)
- before
![before-panel](https://github.com/user-attachments/assets/603d25ba-72bb-4b0f-addf-557847534be5)
- after
![after-panel](https://github.com/user-attachments/assets/1067afc5-ba31-4029-ae71-9975c31d6258)

Tags: 
- before
![before](https://github.com/user-attachments/assets/9d5e4891-82ce-493e-851a-7d3893124996)
- after
![after](https://github.com/user-attachments/assets/56b638ef-c1d8-4cda-9d39-9cb8c3348ad8)

Table: 
![before](https://github.com/user-attachments/assets/6e24b372-5395-46fd-9ba6-3c81e2a8cd79)
![after](https://github.com/user-attachments/assets/a55bfa18-ec36-443c-954c-7ba0c8d3d5ed)
![before-2](https://github.com/user-attachments/assets/e9295894-db24-4e33-bda0-5930dc878b56)
![after-2](https://github.com/user-attachments/assets/a36d1798-2d9b-4343-a5b4-1e06b1bb64bb)
![before](https://github.com/user-attachments/assets/575d0d9e-12db-4de6-96c9-49e6d10f67f6)
![after](https://github.com/user-attachments/assets/9bee9d4c-20e1-4d83-a3e2-e64590df86a2)
![before](https://github.com/user-attachments/assets/70639d6f-a0da-43e0-8536-94e53e7a333b)
![after](https://github.com/user-attachments/assets/51751819-0b0c-46b1-a0d2-033e5503a040)

Trash:
![before](https://github.com/user-attachments/assets/efbd5d7b-ab99-4032-97fc-ae09e0a29563)
![after](https://github.com/user-attachments/assets/f7f72ef6-f4d8-49ff-bfb7-cad5d9d67423)
![before](https://github.com/user-attachments/assets/e565b348-250c-46ca-a0c6-fe89040bd546)
![after](https://github.com/user-attachments/assets/c6600af2-2033-492b-8c9b-56adf8b191f7)
![before](https://github.com/user-attachments/assets/12e6c814-43c1-4a0a-9381-acbcbfd0bbc1)
![after](https://github.com/user-attachments/assets/aeff0acc-8ceb-4450-8707-3238a37fd64a)
![before](https://github.com/user-attachments/assets/c01d3510-79c0-4fef-a13f-284269cea09b)
![after](https://github.com/user-attachments/assets/cfa83c0d-2cea-4d73-941b-9108705825ea)
![before](https://github.com/user-attachments/assets/01fe8536-7dd8-4e32-b2d2-6d601d13d088)
![after](https://github.com/user-attachments/assets/86203c1a-856e-4a8b-9ee5-5d4152ca857d)
![before](https://github.com/user-attachments/assets/5fd234d7-de04-4abc-9305-cc82ef842a27)
![after](https://github.com/user-attachments/assets/2b9c7b4f-ae40-45a3-8776-9e6ed33f89b0)
![before](https://github.com/user-attachments/assets/7d48fa7e-a242-46d4-9d3e-237b5716179e)
![after](https://github.com/user-attachments/assets/6cb687c2-f7a9-4033-8f9f-7a162eaba81e)
![before](https://github.com/user-attachments/assets/83c767dd-7636-4d98-9b46-55d55bb10fd3)
![after](https://github.com/user-attachments/assets/334e45cb-71e5-44b9-82f6-f5e6fae11470)

Link: 

- before
![before](https://github.com/user-attachments/assets/485af984-39ef-4d39-8766-05c189d38c50)
- after
![after](https://github.com/user-attachments/assets/568bcb6c-c17d-4819-b65d-6ffb1f37ee29)
### TESTING INSTRUCTIONS
<!--- Required! W
hat steps can be taken to manually verify the changes? -->
1. CI should pass
2. All custom icons replaced with Ant Design 5 icons, should be consistent in color, size, and relevance.

### ADDITIONAL INFORMATION
<!--- Check any relevant boxes with "x" -->
<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

EnxDev committed:
replaces custom icons (trash, link) with antd5 icons

EnxDev committed:
refactor(icons): Replaces custom icons (table, table-chart-title, dataset-physical) with antd5 icons

EnxDev committed:
refactor(icons): Replaces custom icons (tags, filter) with antd5 icons

EnxDev committed:
refactor(icons): Replaces custom icons (LockUnlocked, LockLocked, CardView, ListView, CaretRight, CaretUp, CaretDown) with antd5 icons

EnxDev committed:
refactor(icons): Replaces custom icons (Plus, PlusCircle) with antd5 icons

EnxDev committed:
refactor(caretDown):
- Small adjustments for caret down icon
- Restored menu icons in the header menu

EnxDev committed:
refactor(icons): Replaces custom icon (MoreVert) with antd5 icon (MoreOutlined)

EnxDev committed:
refactor(icons):
- Replaces custom icon (Refresh) with antd5 icon (SyncOutlined)
- Small icons position adjustments

EnxDev committed:
wip(close icon): Replaces close custom icon with antd5 CloseOutlined icon

EnxDev committed:
refactor(icons): Replaces custom icons (fa-x, xSmall, xLarge) with antd5 icon (CloseOutlined)

EnxDev committed:
refactor(icons):
- Migrates custom icons (Check, Calendar, FavoriteSelected, FavoriteUnselected, TableOutlined, Alert, AlertSolid, Eye, AlertSolid) to Ant Design 5 icons
- Adds SaveOutlined icon to the dashboard save button
- Small alignment adjustments for icons
- Changes the order of buttons in the chart dropdown on the homepage
- Adds custom CircleSolid icon
- wip SqlEditorTabHeader

EnxDev committed:
refactor(icons):
- Replaces custom icons (Check, CheckCircle, CheckSolid, Clock, Minus, FieldDerived, Sql, fa-database) with antd5 icon (CloseOutlined)
- Removes TabStatusIcon component

geido commented on superset-frontend/src/SqlLab/components/SqlEditorTabHeader/index.tsx:
```diff
@@ -146,7 +182,14 @@ const SqlEditorTabHeader: FC<Props> = ({ queryEditor }) => {
       />
-      <TabTitle>{qe.name}</TabTitle> <TabStatusIcon tabState={queryState} />{' '}
+      <TabTitle>{qe.name}</TabTitle>
+      {StatusIcon && (
```
We don't need this check

geido commented on superset-frontend/src/SqlLab/components/SqlEditorTabHeader/index.tsx:
```diff
@@ -146,7 +182,14 @@ const SqlEditorTabHeader: FC<Props> = ({ queryEditor }) => {
+      <TabTitle>{qe.name}</TabTitle>
+      {StatusIcon && (
+        <StatusIcon
+          css={{ margin: '0px!important' }}
```
Let's try to remove this !important if possible

EnxDev committed:
refactor(icons): Replaces custom icons (Search, Email, Share) with antd5 icon

EnxDev committed:
Merge branch 'master' into enxdev/refactor-antd5-icons

EnxDev committed:
refactor(icons):
- Migrates some icons (Import, EditAlt, Edit, Download, Collapse, Expand, more_horiz, Copy)  from custom icons to Antd5 icons
- Remove some '!important' from CSS
- Small alignment adjustments for icons
- Adds the edit icon next to the title in DatasourceModal
- Adds the edit icon next to the title in PropertiesModal

EnxDev committed:
refactor(icons): Replaces custom icons (NavDashboard, NavChart, error, error_solid, warning, warning_solid, info, info-solid, info_solid_small, sort_asc, sort_desc, sort, fa fa-key, field_boolean, field_abc) with antd5 icons

EnxDev committed:
lint

EnxDev committed:
Merge branch 'master' into enxdev/refactor-antd5-icons

geido commented on superset-frontend/src/SqlLab/components/ResultSet/index.tsx:
```diff
@@ -677,7 +688,8 @@ const ResultSet = ({
                         overflow: hidden;
                         white-space: nowrap !important;
-                        text-overflow: ellipsis;
+                        text-overflow: ellipsimport { CopyToClipboard } from '.';
```
You might want to check this

geido commented on superset-frontend/src/SqlLab/components/SqlEditorTabHeader/index.tsx:
```diff
@@ -31,11 +37,17 @@ import {
+
+  [aria-label='check-circle'],
+  .status-icon {
+    margin: 0px;
+    margin-right: 0px;
```
Do we need both?

geido commented on superset-frontend/src/components/Datasource/CollectionTable.tsx:
```diff
@@ -152,6 +152,9 @@ const StyledButtonWrapper = styled.span`
     margin-top: ${theme.gridUnit * 3}px;
     margin-left: ${theme.gridUnit * 3}px;
+    button>span>:first-of-type {
+      margin-right:0;
```
```suggestion
      margin-right: 0;
```

geido commented on superset-frontend/src/components/Datasource/DatasourceEditor.jsx:
```diff
@@ -971,9 +974,21 @@ class DatasourceEditor extends PureComponent {
+              <Icons.UnlockOutlined
+                iconSize="xl"
+                iconColor={theme.colors.grayscale.base}
+                css={theme => ({
+                  margin: `auto ${theme.gridUnit}px auto 0`,
+                })}
```
```suggestion
                css={(theme) => css`
                  margin: auto ${theme.gridUnit}px auto 0;
               `}
```

Can we do this everywhere for consistency with the rest of the codebase? Just import `css` in the file.

geido commented on superset-frontend/src/components/Datasource/DatasourceEditor.jsx:
```diff
@@ -971,9 +974,21 @@ class DatasourceEditor extends PureComponent {
+              <Icons.LockOutlined
+                iconSize="xl"
+                iconColor={theme.colors.grayscale.base}
+                css={theme => ({
+                  margin: `auto ${theme.gridUnit}px auto 0`,
+                })}
```
```suggestion
                css={(theme) => css`
                  margin: auto ${theme.gridUnit}px auto 0;
               `}
```

geido commented on superset-frontend/src/components/Datasource/DatasourceModal.tsx:
```diff
@@ -298,6 +301,11 @@ const DatasourceModal: FunctionComponent<DatasourceModalProps> = ({
         <span>
+          <Icons.EditOutlined
+            iconColor={theme.colors.grayscale.base}
+            css={{ margin: `auto ${theme.gridUnit * 2}px auto 0` }}
```
Same as my other comments

EnxDev committed:
- Set default color for icons and removes it where it's explicitly setted in the codebase.
- Minor adjustments to address review comments.

EnxDev committed:
Antd4 icons migration to Antd5 icons

EnxDev committed:
Merge branch master into enxdev/refactor-antd5-icons

EnxDev committed:
Merge branch 'master' into enxdev/refactor-antd5-icons

EnxDev committed:
fix InfoTooltip

EnxDev committed:
Merge branch 'master' into enxdev/refactor-antd5-icons

EnxDev committed:
fix cypress action test

EnxDev committed:
- wip(Icon): prevent double <span> containing the icon
- Update shim to pass the component
- Small fix for icon alignment

EnxDev committed:
remove some css

EnxDev committed:
refactor(Icon): wip remove double span that contain custom icon

EnxDev committed:
Merge branch 'master' into enxdev/refactor-antd5-icons

msyavuz commented on superset-frontend/src/features/charts/ChartCard.tsx:
```diff
@@ -68,10 +68,39 @@ export default function ChartCard({
-
   const menu = (
     <Menu>
+      {canEdit && (
+        <Menu.Item>
+          <div
+            data-test="chart-list-edit-option"
+            role="button"
+            tabIndex={0}
+            onClick={() => openChartEditModal(chart)}
+          >
+            <Icons.EditOutlined
+              iconSize="l"
+              css={{ verticalAlign: 'baseline' }}
+            />{' '}
+            {t('Edit')}
+          </div>
+        </Menu.Item>
+      )}
+      {canExport && (
+        <Menu.Item>
+          <div
+            role="button"
+            tabIndex={0}
+            onClick={() => handleBulkChartExport([chart])}
+          >
+            <Icons.UploadOutlined
+              iconSize="l"
+              css={{ verticalAlign: 'baseline' }}
+            />{' '}
+            {t('Export')}
+          </div>
+        </Menu.Item>
+      )}
```
Is this order change intentional?

EnxDev replied:
It makes coherence with the others

EnxDev committed:
remove some unused const theme

EnxDev committed:
lint

EnxDev commented on superset-frontend/src/components/DropdownSelectableIcon/index.tsx:
```diff
@@ -0,0 +1,177 @@
+      onVisibleChange={handleVisibleChange}
+    />
+  );
+};
```
I don’t know where it came from; maybe it's a regression. I remove that.

EnxDev committed:
remove DropdownSelectableIcon component

geido commented on superset-frontend/src/components/LastUpdated/index.tsx:
```diff
@@ -49,14 +49,16 @@ const TextStyles = styled.span`
-  color: ${({ theme }) => theme.colors.primary.base};
+const RefreshIcon = styled(Icons.SyncOutlined)`
+  ${({ theme }) => `
+  color: ${theme.colors.primary.base};
```
Should this color be used as a prop of the icon?

EnxDev replied:
Same here, I think we should use the default color that is inside Icon component

geido commented on superset-frontend/src/explore/components/controls/DatasourceControl/index.jsx:
```diff
@@ -110,6 +110,7 @@ const Styles = styled.div`
     overflow: hidden;
   }
   .datasource-svg {
+    color: ${({ theme }) => theme.colors.grayscale.base};
```
Can we use this as a prop of the icon?

EnxDev replied:
I think we can use the default color inside the Icon component if we're referring to the icon.

geido commented on superset-frontend/src/pages/CssTemplateList/index.tsx:
```diff
@@ -196,7 +198,15 @@ function CssTemplatesList({
+            iconColor={theme.colors.primary.light5}
+            iconSize="s"
+            css={css`
+              margin: 'auto ${theme.gridUnit * 2}px auto 0';
+              vertical-align: 'baseline';
```
I think you don't need to use quoting here. You might want to check others too

```suggestion
              margin: auto ${theme.gridUnit * 2}px auto 0;
              vertical-align: baseline;
```

geido commented on superset-frontend/src/pages/DatasetList/index.tsx:
```diff
@@ -624,7 +632,15 @@ const DatasetList: FunctionComponent<DatasetListProps> = ({
+            iconSize="s"
+            css={css`
+              margin: auto ${theme.gridUnit * 2}px auto 0;
+              verticalalign: baseline;
```
```suggestion
              vertical-align: baseline;
```

geido reviewed (commented):
Overall looking good. Thanks for all the work on this. There are a few nits and some style guidelines to check (such as the use of tagged template literals. I will do some functional testing next.

EnxDev committed:
wip Icon component

EnxDev commented on superset-frontend/src/components/Icons/Icon.tsx:
```diff
@@ -48,6 +70,7 @@ export const StyledIcon = styled(AntdIconComponent)<IconType>`
 
 export interface IconProps extends IconType {
   fileName: string;
+  customIcons?: boolean;
```
This component is under construction. With this prop, I want to avoid the double <span> for the icon.

geido commented on superset-frontend/src/dashboard/components/URLShortLinkButton/index.tsx:
```diff
@@ -88,7 +89,10 @@ export default function URLShortLinkButton({
             copyNode={
-              <i className="fa fa-clipboard" title={t('Copy to clipboard')} />
+              <>
+                <Icons.CopyOutlined iconSize="m" />
```
Copy to clipboard does not work for me right now. Is the issues related to these changes? 
<img width="712" alt="Screenshot 2025-02-26 at 17 30 43" src="https://github.com/user-attachments/assets/bd5528fe-dbe5-4328-8deb-34da0827ce76" />

EnxDev replied:
Good point, I'll fix that!

EnxDev committed:
lint

EnxDev committed:
Fix almost all review comments

EnxDev committed:
clean up

alexandrusoare commented on superset-frontend/src/components/Icons/Icon.tsx:
```diff
@@ -73,8 +101,8 @@ export const Icon = (props: IconProps) => {
-
   return (
+    // @ts-ignore to be removed
     <StyledIcon
```
Just a small nit, but in storybook the order of the sizes is ['s', 'l', 'm', 'xl', 'xxl'] and it should be ['s', 'm', 'l', 'xl', 'xxl']

alexandrusoare commented:
There's this icon which on master get's the color blue, and after upgrading it keeps its color. Is this the intended behavior? 
BEFORE
![image](https://github.com/user-attachments/assets/9563955f-49e4-4bdd-b673-6db6778d3e94)
AFTER
![image](https://github.com/user-attachments/assets/ff27d6a5-c6db-484a-a059-e528890f782b)

EnxDev commented:
@alexandrusoare Thanks for your comments! We have decided to set up a default color for the icons. 

These icons probably don’t have the assigned colors.

When I’ve finished the remaining work, I’ll take a look at it. Thanks!

kgabryje commented:
> Here as well: BEFORE: ![image](https://private-user-images.githubusercontent.com/37236580/417529400-f41ebb00-5c0b-46c1-a98a-797da51beb3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA2NTI0MTUsIm5iZiI6MTc0MDY1MjExNSwicGF0aCI6Ii8zNzIzNjU4MC80MTc1Mjk0MDAtZjQxZWJiMDAtNWMwYi00NmMxLWE5OGEtNzk3ZGE1MWJlYjNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI3VDEwMjgzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY3ZmVlNTYwYTQzZDYwZWNmYjk5MzEyNWRlNjY0YzIzMjNjNzdhMGFhMGVhMGM5ZTBkMDc3NmNiNWI2MjM3MGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.QtUktr-G8W0zlzFh1ULECbch3Jb6VtHY24K53U-ysK8) AFTER: ![image](https://private-user-images.githubusercontent.com/37236580/417529561-750e2efb-3607-4ebb-8652-9cd56f967135.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA2NTI0MTUsIm5iZiI6MTc0MDY1MjExNSwicGF0aCI6Ii8zNzIzNjU4MC80MTc1Mjk1NjEtNzUwZTJlZmItMzYwNy00ZWJiLTg2NTItOWNkNTZmOTY3MTM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjI3VDEwMjgzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkyMmMxOWZiN2IyNmZiYWZhZTVlMmU1ZTY1MWUzNjljYzg2ZDM3M2QwNDA1NTBhOTE2OTIwZWRlZmM2YmJkMzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FWL0oxKw_UPWZVwXzzDqN1AvtNwb8tDuNifOXOIj-ao)

It also looks like the icons are not centered in the new version

EnxDev committed:
fix(filter): change the color of the icons and fix the vertical alignment of the icons

EnxDev committed:
refactor: add color to some icons

EnxDev committed:
fix(SliceAdder): type adjustments

EnxDev committed:
lint

EnxDev committed:
fix cy actions test

EnxDev committed:
fix cy filter test

EnxDev committed:
lint

EnxDev committed:
fix change some icons size

EnxDev committed:
fix(menu): align "SQL" menu item vertically and replace triangle-down icon
- Align "SQL" menu item vertically
- Replace `triangle-down` icon with `CaretDownOutlined`

EnxDev committed:
test(filters): fix Cypress tests for nativeFilters.noInitState and filter tests

EnxDev committed:
Merge branch 'master' into enxdev/refactor-antd5-icons

EnxDev committed:
fix some RTL tests

EnxDev committed:
fix RTL tests

EnxDev committed:
fix some tests

EnxDev committed:
fix some tests

EnxDev committed:
style(Icons): Change size and color for some Icons

EnxDev committed:
fix(RTL): update RTL and Cypress tests to check replaced icons

EnxDev committed:
Merge branch master into enxdev/refactor-antd5-icons

EnxDev committed:
lint

EnxDev committed:
fix cy test

geido committed:
chore(Icon): Refactor and enhance

EnxDev commented on superset-frontend/cypress-base/cypress/e2e/dashboard/actions.test.js:
```diff
@@ -24,21 +24,44 @@ describe('Dashboard actions', () => {
     cy.getBySel('dashboard-header-container')
-      .find("[aria-label='favorite-selected']")
-      .click();
+      .find("[aria-label='star']")
```
I think we should distinguish star filled from star outlined

geido committed:
chore(Icons): Remove direct antdesign imports

EnxDev committed:
fix(BaseIcon): correct CSS interpolation with Emotion

EnxDev committed:
fix(URLShortLinkButton): restore copy functionality

EnxDev committed:
fix close icon on SQL lab

geido commented on superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('dashboard list view', () => {
+    cy.get('[aria-label="appstore"]').click();
     cy.eyesOpen({
-      testName: 'Dashboards card-view',
+      testName: 'Dashboards AppstoreOutlined',
```
```suggestion
      testName: 'Dashboards card-view',
```

geido commented on superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('dashboard list view', () => {
+    cy.get('[aria-label="unordered-list"]').click();
     cy.eyesOpen({
-      testName: 'Dashboards list-view',
+      testName: 'Dashboards UnorderedListOutlined',
```
```suggestion
      testName: 'Dashboards list-view',
```

geido commented on superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('dashboard list view', () => {
+      testName: 'Dashboards UnorderedListOutlined',
     });
-    cy.eyesCheckWindow('Dashboards list-view loaded');
+    cy.eyesCheckWindow('Dashboards UnorderedListOutlined loaded');
```
```suggestion
    cy.eyesCheckWindow('Dashboards list-view loaded');
```

geido commented on superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('charts list view', () => {
+      testName: 'Charts AppstoreOutlined',
     });
-    cy.eyesCheckWindow('Charts card-view loaded');
+    cy.eyesCheckWindow('Charts AppstoreOutlined loaded');
```
```suggestion
    cy.eyesCheckWindow('Charts card-view loaded');
```

geido commented on superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('charts list view', () => {
+    cy.get('[aria-label="appstore"]').click();
     cy.eyesOpen({
-      testName: 'Charts card-view',
+      testName: 'Charts AppstoreOutlined',
```
```suggestion
      testName: 'Charts card-view',
```

geido commented on superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('charts list view', () => {
+      testName: 'Charts UnorderedListOutlined',
     });
-    cy.eyesCheckWindow('Charts list-view loaded');
+    cy.eyesCheckWindow('Charts UnorderedListOutlined loaded');
```
```suggestion
    cy.eyesCheckWindow('Charts list-view loaded');
```

geido commented on superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('charts list view', () => {
+    cy.get('[aria-label="unordered-list"]').click();
     cy.eyesOpen({
-      testName: 'Charts list-view',
+      testName: 'Charts UnorderedListOutlined',
```
```suggestion
      testName: 'Charts list-view',
```

geido commented on superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts:
```diff
@@ -28,18 +28,18 @@ describe('dashboard list view', () => {
+      testName: 'Dashboards AppstoreOutlined',
     });
-    cy.eyesCheckWindow('Dashboards card-view loaded');
+    cy.eyesCheckWindow('Dashboards AppstoreOutlined loaded');
```
```suggestion
    cy.eyesCheckWindow('Dashboards card-view loaded');
```

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/chartlist.test.ts

geido committed:
Update superset-frontend/cypress-base/cypress/applitools/dashboardlist.test.ts

EnxDev committed:
style: change some icon colors and alignments

geido committed:
chore(Icons): Adjust sizes / paddings

geido committed:
Merge branch 'enxdev/refactor-antd5-icons' of https://github.com/apache/superset into enxdev/refactor-antd5-icons

geido committed:
Merge branch 'master' of https://github.com/apache/superset into enxdev/refactor-antd5-icons

geido committed:
chore(Icons): Sync with master

kasiazjc commented:
Thank yo so much @EnxDev ❤️ 

A few things I found so far 


- Action icons in CRUD views are 24x24px and should be 16x16px for a more balanced look (for example favorites star is 16x16)
<img width="1293" alt="image" src="https://github.com/user-attachments/assets/985f6da1-ce01-4f2b-a350-a28d8bfaa18f" />


- Icons in buttons should be bigger (10px is not really visible), I can see in ant-d they are 14x14px and we can follow that for keeping our implementation pretty much vanilla without lots of deviations. I can see that on master we use in here some weird size like 10x12, but the previous icon was much more bold, so it needs to be bigger. NOTE: I found that in some buttons we use 16px (for example save button in explore), so not sure what the rule is. Is this the correct time to update it? @EnxDev @geido 
<img width="373" alt="image" src="https://github.com/user-attachments/assets/84a96c95-8fb1-4628-8d57-0ca9c8a57d54" />

- might be old implementation, but this icon should match other button icons like mentioned above (14px)
<img width="335" alt="image" src="https://github.com/user-attachments/assets/15aaac66-3428-4975-b396-b039a7359f9d" />



- For somer reasons in the navbar spacings between caret-down and text/icons disappeared. Can you add at least 4px spacings? 

now:
<img width="949" alt="image" src="https://github.com/user-attachments/assets/5800be7a-281c-4c73-b88a-2a205f7d818c" />

before:
<img width="949" alt="image" src="https://github.com/user-attachments/assets/9a58d041-6e0c-416b-af95-2ce7f301bb1e" />

- Controls in control panel - in the new implementation the caret-right is filled, and should be just an empty arrow like currently 

now: 
<img width="302" alt="image" src="https://github.com/user-attachments/assets/ccb0b01f-d5e8-4b51-b394-32fc9e728c80" />

before:
<img width="302" alt="image" src="https://github.com/user-attachments/assets/93d1c7d9-06b7-45be-94a6-adc253014af9" />

- in dashboard filters panel (vertical) sizing between gear and collapse icon got bigger than before. It should be 8px

now: 
<img width="258" alt="image" src="https://github.com/user-attachments/assets/46a6ac5b-411f-427c-b791-2b33a91af1e6" />

before:
<img width="258" alt="image" src="https://github.com/user-attachments/assets/d8166a20-efec-4c68-81c0-88a7e811f80b" />


- Icon in dataset settings should be 16px for better balance

<img width="972" alt="image" src="https://github.com/user-attachments/assets/d9a6484b-4615-46da-a15d-aa7ab7802354" />

- not sure if this icon should be new too, but pencil doesnt seem to be updated to our current edit icon (like above) 

<img width="972" alt="image" src="https://github.com/user-attachments/assets/e03b3236-0e43-465a-9d74-2e11bb41f2cc" />

- this caret (collapse/expand) icon on the right should match icons on the left (arrow not filled)

<img width="725" alt="image" src="https://github.com/user-attachments/assets/2cc67b78-1250-40e1-b3c2-35bdc3cc08ca" />

<img width="972" alt="image" src="https://github.com/user-attachments/assets/3cd964db-11e8-47ff-b506-98791b0eb66d" />

geido committed:
chore(DropdownButton): Implement divider for default dropdown buttons

EnxDev committed:
Merge branch master into enxdev/refactor-antd5-icons

EnxDev committed:
style(action icons): change size for action icons

EnxDev committed:
style(buttons): increase icon size inside the buttons

EnxDev committed:
style(icon): adjust margin and replace fa-pencil, CaretDownOutlined, CaretRightOutlined

EnxDev commented:
> Thank yo so much @EnxDev ❤️

@kasiazjc Thanks for your review as well ❤️
> 
> A few things I found so far
> 
> * Action icons in CRUD views are 24x24px and should be 16x16px for a more balanced look (for example favorites star is 16x16) DONE ✅
> 
> <img alt="image" width="1293" src="https://private-user-images.githubusercontent.com/36897697/420383908-985f6da1-ce01-4f2b-a350-a28d8bfaa18f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODM5MDgtOTg1ZjZkYTEtY2UwMS00ZjJiLWEzNTAtYTI4ZDhiZmFhMThmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQyYmVmMDkzMTI5OGE0MmM1M2VlYmJkOTk5YjE0NzlhNmEwYTNkMWZmYmMwMzVmZjMxNTFjZGI4Yjg1MTVlMTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qD7R83LrugnhDZdIYMbsArswZCqldAi1NCORiqGHVcc">
> * Icons in buttons should be bigger (10px is not really visible), I can see in ant-d they are 14x14px and we can follow that for keeping our implementation pretty much vanilla without lots of deviations. I can see that on master we use in here some weird size like 10x12, but the previous icon was much more bold, so it needs to be bigger. NOTE: I found that in some buttons we use 16px (for example save button in explore), so not sure what the rule is. Is this the correct time to update it? @EnxDev @geido
DONE ✅
> 
> <img alt="image" width="373" src="https://private-user-images.githubusercontent.com/36897697/420385184-84a96c95-8fb1-4628-8d57-0ca9c8a57d54.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODUxODQtODRhOTZjOTUtOGZiMS00NjI4LThkNTctMGNhOWM4YTU3ZDU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwMWYxYWY0OWJmMDdhZjNjZjUyNGNmOTRkNDZlNjMyYTEwZDJiNGM5NzZlNDI0ZjUxNDNjMzhjMTI2ZjI0ZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.aUyibfyXtZJoX2sGqpfURz2cOG-wP3MloBXf5_zCGsg">
> * might be old implementation, but this icon should match other button icons like mentioned above (14px)
DONE ✅
> 
> <img alt="image" width="335" src="https://private-user-images.githubusercontent.com/36897697/420394656-15aaac66-3428-4975-b396-b039a7359f9d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzOTQ2NTYtMTVhYWFjNjYtMzQyOC00OTc1LWIzOTYtYjAzOWE3MzU5ZjlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdlZTFlODA3MWRjOTBhMWRmZmQwNzk0NGFkMWNmMzk3OTRmN2RlNWJjNTgyMTBlNmVlOGEzMDcxMDU4MDMyOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1owdJANzDgKbC6xFXfC5EAfhBx70wlhf_DxOxb5KPng">
> * For somer reasons in the navbar spacings between caret-down and text/icons disappeared. Can you add at least 4px spacings?
DONE ✅
> 
> now: <img alt="image" width="949" src="https://private-user-images.githubusercontent.com/36897697/420386914-5800be7a-281c-4c73-b88a-2a205f7d818c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODY5MTQtNTgwMGJlN2EtMjgxYy00YzczLWI4OGEtMmEyMDVmN2Q4MThjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4NjMyYjkwM2FjMTgzZTMwZGFmNjRiZWRmNDMxZDY0NzJiYzZkODQwYzNiNDhlOTk4M2EyZDQ1ODExYmYzMjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WH-3IM2ymO3MRawd2sbZU4Kl6i5hOgsnnK9ERp9sgPc">
> 
> before: <img alt="image" width="949" src="https://private-user-images.githubusercontent.com/36897697/420387026-9a58d041-6e0c-416b-af95-2ce7f301bb1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODcwMjYtOWE1OGQwNDEtNmUwYy00MTZiLWFmOTUtMmNlN2YzMDFiYjFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEwYjljNDNjYjY2NTA1YjQ2N2VhNjZjYjUxZTk2NjBjMzFmZmE5M2I2NjA0ZjVmZTI3NWZhOTQyNGRiMGQ3ZmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.zpDi6kNdAfjKVt4SXfRBzIDz1lu4gqG80W4uag8Jayc">
> 
> * Controls in control panel - in the new implementation the caret-right is filled, and should be just an empty arrow like currently
DONE ✅
> 
> now: <img alt="image" width="302" src="https://private-user-images.githubusercontent.com/36897697/420387574-ccb0b01f-d5e8-4b51-b394-32fc9e728c80.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODc1NzQtY2NiMGIwMWYtZDVlOC00YjUxLWIzOTQtMzJmYzllNzI4YzgwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0NmZmZWRhMDk1Y2E1ZWEzZjFmYzZjYzdiNWFkMzFjZGJmNGNhMzM5MmFiNTI3ZTM0NmMzZjAwM2NiNjExZjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qlKkolRgiW8PYW94PL6zVJDOEQN6jUbglSIxHmOClc4">
> 
> before: <img alt="image" width="302" src="https://private-user-images.githubusercontent.com/36897697/420387762-93d1c7d9-06b7-45be-94a6-adc253014af9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODc3NjItOTNkMWM3ZDktMDZiNy00NWJlLTk0YTYtYWRjMjUzMDE0YWY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTljNmQyMmQ0NzYzZWRkZGFiZWQ3YzY0MWM5ZTU0N2VjODE5YTNmNzE4ZWIwZDdlNTcxNTc2MGEzZWI0YTMxYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GdJVAyeplTGUGa8ZPTWStFrXsywC0Fqq4ZZyZkzZyXA">
> 
> * in dashboard filters panel (vertical) sizing between gear and collapse icon got bigger than before. It should be 8px DONE ✅
> 
> now: <img alt="image" width="258" src="https://private-user-images.githubusercontent.com/36897697/420389494-46a6ac5b-411f-427c-b791-2b33a91af1e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODk0OTQtNDZhNmFjNWItNDExZi00MjdjLWI3OTEtMmIzM2E5MWFmMWU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4ODQzMWQ2ODVhNmNlYzc1YzMzNTFkODA4ZWJlZjYyZTM4OTUzNGNlYmRiOTg5ZTE4ODQ5MzcxZDUyODM0ZWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2uesDwh0PsEzyfjxEWE4i5DgdUmOUAWaTFvEC_qnYGQ">
> 
> before: <img alt="image" width="258" src="https://private-user-images.githubusercontent.com/36897697/420389344-d8166a20-efec-4c68-81c0-88a7e811f80b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzODkzNDQtZDgxNjZhMjAtZWZlYy00YzY4LTgxYzAtODhhN2U4MTFmODBiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0ODcyNDFhYjllZjRhNTY3MDZjYjFkMjQ1MjI5YzAyYjVkMWUwOWRiMGQ5ZTFhMWZjYmYzZDQ3OTA2ZjhlZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5w2UUFk77Ou5TRPn6XDNY_v0HKdzZA4JCol8QORGu7w">
> 
> * Icon in dataset settings should be 16px for better balance DONE ✅
> 
> <img alt="image" width="972" src="https://private-user-images.githubusercontent.com/36897697/420392649-d9a6484b-4615-46da-a15d-aa7ab7802354.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzOTI2NDktZDlhNjQ4NGItNDYxNS00NmRhLWExNWQtYWE3YWI3ODAyMzU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ2OTRlMDA1ZTdlYTlkZTI4OGZlNmRmNDcyNGJkOGNjMjhmNWMwZGFjOTAwODk3MzM3YmExMzczNjcxZjRjNTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.aNTeI3WiBK1zZMNokbNQNtv_joK-NgdHh5ZM2P-seJM">
> * not sure if this icon should be new too, but pencil doesnt seem to be updated to our current edit icon (like above) DONE ✅
> 
> <img alt="image" width="972" src="https://private-user-images.githubusercontent.com/36897697/420392981-e03b3236-0e43-465a-9d74-2e11bb41f2cc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzOTI5ODEtZTAzYjMyMzYtMGU0My00NjVhLTlkNzQtMmUxMWJiNDFmMmNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcwZjljMGYxY2RkYTVmZDQwMTg5Mzk2MGY4Y2U0ZjhhMWEyMjc2NzcyNjJmMThiMzMwMzAyNzc4ODE3M2M3NjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.65XPV0uOlwzrxDj328bpQPrkK_b9Mu5rpDHlTkMd6EA">
> * this caret (collapse/expand) icon on the right should match icons on the left (arrow not filled) DONE ✅
> 
> <img alt="image" width="725" src="https://private-user-images.githubusercontent.com/36897697/420393612-2cc67b78-1250-40e1-b3c2-35bdc3cc08ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzOTM2MTItMmNjNjdiNzgtMTI1MC00MGUxLWIzYzItMzViZGMzY2MwOGNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM3MWJlMjM3Zjc0MWViMDg4OTQ0OGM1ODdkZmFkN2IxNDk3YTJiMmVkYzg4N2YzMDE1NTg1MDIzMWI3ZjE4MjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.bmnc0ixxXSHbt-AIePkiWICrBHQQcWab4I4RSmBJUOA"> <img alt="image" width="972" src="https://private-user-images.githubusercontent.com/36897697/420393454-3cd964db-11e8-47ff-b506-98791b0eb66d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIwNTUyMTksIm5iZiI6MTc0MjA1NDkxOSwicGF0aCI6Ii8zNjg5NzY5Ny80MjAzOTM0NTQtM2NkOTY0ZGItMTFlOC00N2ZmLWI1MDYtOTg3OTFiMGViNjZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE1VDE2MDgzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIzNWRmYWIyZTNiY2E2Y2NjM2RjMTgzYmVkMzg3ZWExYjhjOThmOTYxODUzOTU2YjVkNDdmZjY0ZTY1NzEwMzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HcnnvYmf3bCLQ11ds6-27y3JXxA0Pvt38wNlCey1zws">

EnxDev committed:
fix RTL tests

EnxDev committed:
refactor(icons): remove unused custom icons

EnxDev committed:
Merge branch master into enxdev/refactor-antd5-icons

EnxDev committed:
Re-add deleted icons: transparent, error_solid_small_red, and error

EnxDev committed:
refactor: remove some fa fa icons, add custom icons files

EnxDev committed:
fix: add missing license for square icon

EnxDev committed:
fix rtl tests

EnxDev committed:
lint

geido committed:
Merge branch 'enxdev/refactor-antd5-icons' of https://github.com/apache/superset into enxdev/refactor-antd5-icons

geido committed:
chore(Icons): Improve alignments and minor fixes

mistercrunch commented:
Looks like there's a bit of lint left, highly recommend installing pre-commit if you haven't already

EnxDev committed:
Merge branch master into enxdev/refactor-antd5-icons

EnxDev committed:
lint
